homeassistant/components/balboa/translations/nb.json

```diff
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Koble til Balboa Wi-Fi enheten",
+        "data": {
+          "host": "Vert",
+          "name": "Navn"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "Kunne ikke koble til, prøv igjen",
+      "unknown": "Uventet feil"
+    },
+    "abort": {
+      "already_configured": "Enheten er allerede konfigurert"
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "sync_time": "Hold din Balboa Spa Client tid synkronisert med Home Assistant"
+        }
+      }
+    }
+  }
+}
```

Comment:
Only en.json should be uploaded (other languages are added automatically by the translator)
```suggestion
```